core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -216,6 +216,8 @@ object AppType {
 }
 
 final case class ReleaseName(value: String) extends AnyVal
+final case class RemoteUserName(value: String) extends AnyVal
```

Comment:
just for my own understanding, what will be the value for this remote user name and user secret? and will they be the same for all users and stored in vault?

Reply:
I was planning to just send `galaxy-user` as the user and `galaxy-secret` as the secret for all users. However this isn't really working on the Galaxy end currently so the approach may change.

Good question on vault -- I wasn't really planning to store the secret in vault but it may be a good idea..

Reply:
I see...if it's just dummy value, probably not worth storing in vault then..I thought this will be actual secret